pkg/kubernetes/events/event_pubsub_test.go

```diff
@@ -21,15 +21,6 @@ func TestPubSubEvents(t *testing.T) {
 		publish       PubSubMessage
 		expectMessage bool
 	}{
-		{
```

Comment:
Is the CRD equivalent already there for this test or does it need to be added while this is being removed?

Reply:
`client_test.go` -> `TestMeshConfigEventTriggers`

Reply:
Thanks Edu! Since the equivalent already exists, I will go ahead and delete this